### ISSUE TITLE
NullPointerException in GetPath

### ISSUE THREAD

**Issue by IlyaGulya** [miguelbcr/RxPaparazzo#61]
Hi!
Seems like sometimes we can miss reference to target activity or fragment (i think when we are trying to pass result to detached fragment, for example)
```
Caused by java.lang.NullPointerException
android.provider.DocumentsContract.isDocumentsProvider (DocumentsContract.java:750)
android.provider.DocumentsContract.isDocumentUri (DocumentsContract.java:733)
com.miguelbcr.ui.rx_paparazzo.interactors.GetPath.getPath (GetPath.java:59)
com.miguelbcr.ui.rx_paparazzo.interactors.GetPath.react (GetPath.java:47)
com.miguelbcr.ui.rx_paparazzo.interactors.PickImage$2.response (PickImage.java:62)
rx_activity_result.HolderActivity.onActivityResult (HolderActivity.java:86)
```

**Comment by miguelbcr** [comment]
Hi @IlyaGulya ,
do you know why it's happening?
It seems to be related to #56 in somehow

**Comment by IlyaGulya** [comment]
@miguelbcr Sorry, I've tried to reproduce this issue without success. This is a crash report from Crashlytics.

**Comment by miguelbcr** [comment]
Hi @IlyaGulya,
I've released [version 0.4.5](https://github.com/miguelbcr/RxPaparazzo/releases/tag/0.4.5)

Thanks!
